src/Symfony/Bundle/FrameworkBundle/Command/SecretsRevealCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Secrets\AbstractVault;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
```

Comment:
Is this needed when the class is final?

Reply:
I have no idea. I just copied it from SecretsListCommand (as this command is based on it). 

I don't think it makes much sense on either.

Reply:
@OskarStark That gives us more flexibility 👍 as there is no need to use that class from userland, only the command itself (via CLI) matters

Reply:
Sorry I don't get what you mean.

I can do absolutely the same without the internal tag, the class is final and I can only execute the command in userland

Reply:
I guess what @chalasr means is that we could also rename this class or move it to another namespace without having to worry about consumers of it. Not sure if we really need that flexibility though tbh.

Reply:
Ah ok got it

Reply:
> Not sure if we really need that flexibility though tbh.

👍 Thinking about it, I agree we should not fall in being too much defensive in general.